coordinate.go

```diff
@@ -67,7 +68,7 @@ func (a *Agent) updateCoords(nodeCh <-chan []*api.Node) {
 		}
 
 		if len(nodes) == 0 {
-			a.logger.Printf("[DEBUG] No nodes to probe, will retry in %s", retryTime.String())
+			a.logger.Debug("No nodes to probe", "will retry in ", hclog.Fmt("%s", retryTime.String()))
```

Comment:
In agent.go there is a similar message, what would you think of making this something similar:

```
a.logger.Debug("No nodes to probe, will retry", "time", hclog.Fmt("%s", retryTime.String()))
```